### PR TITLE
[vcpkg] update python3 to 3.7.3 on windows

### PR DESCRIPTION
**Pull request by Rastaban**
- [x] Update the hash
- [ ] Run full rebuild test on Windows triplets

### CODE REVIEW

**Comment by Rastaban** [comment]
Note to self: extra test running on Azure DevOps pipeline #2665207

**Comment by Voskrese** [comment]
maybe better on version 3.6?
many packages work no higher than this version

File: scripts/cmake/vcpkg_find_acquire_program.cmake [hunk 1]
```diff
@@ -79,9 +79,9 @@ function(vcpkg_find_acquire_program VAR)
       set(PROGNAME python)
       set(SUBDIR "python3")
       set(PATHS ${DOWNLOADS}/tools/python/${SUBDIR})
-      set(URL "https://www.python.org/ftp/python/3.5.4/python-3.5.4-embed-win32.zip")
-      set(ARCHIVE "python-3.5.4-embed-win32.zip")
-      set(HASH b5240fdc95088c2d7f65d2dd598650f8dd106b49589d94156bd4a078b108c6cabbe7a38ef73e2b2cf00e8312a93d2e587eac2c54ce85540d3c7a26cc60013156)
+      set(URL "https://www.python.org/ftp/python/3.7.3/python-3.7.3-embed-win32.zip")
+      set(ARCHIVE "python-3.7.3-embed-win32.zip")
+      set(HASH 0)
```
Comment (jherico) [inline scripts/cmake/vcpkg_find_acquire_program.cmake]:
```suggestion
      set(HASH 2c1b1f0a29d40a91771ae21a5f733eedc10984cd182cb10c2793bbd24191a89f20612a3f23c34047f37fb06369016bfd4a52915ed1b4a56f8bd2b4ca6994eb31)
```

**Comment by Rastaban** [comment]
I got distracted for the past few weeks building up the public CI system I am back working on this again.  Doing a full rebuild on windows with 3.7.3 using in [vcpkg-windows-pr:1925](https://dev.azure.com/vcpkg/public/_build/results?buildId=1925).  If lots of ports fail I will fall back to python 3.6 and repeat.

**Comment by Rastaban** [comment]
All builds passed in rebuild test (after rerunning known flaky ports)

**Pull request merged**